test-infrastructure/screenplay/factories/iat-constants.js

```diff
@@ -0,0 +1,126 @@
+export const ACTIVITY_TYPES = [
```

Comment:
@vasgan25 this is the 3rd duplication of this data, to add to the copies in frontend and backend. Can we extract constants to a NPM module?